### ISSUE TITLE
Latest EFI builds won't boot

### ISSUE THREAD

**Issue by maxengel** [libretro/Lakka#524]
With both the x64 and i386 builds from 11/7, a USB stick created with Win32 successfully in Windows 10 only gets to a point where there is a solid cursor in the top-left, and never moves forward.

**Comment by maxengel** [comment]
As additional information, creating the USB stick with Rufus and using UEFI and GPT does not help.

Also, other live images work fine, such as the GParted live drive, also created with Rufus.

**Comment by gouchi** [comment]
Can you check that the [md5](http://sources.lakka.tv/nightly/Generic.x86_64/Lakka-Generic.x86_64-devel-20161106231917-r21640-ge218f0f-efi.img.gz.md5) or [sha256](http://sources.lakka.tv/nightly/Generic.x86_64/Lakka-Generic.x86_64-devel-20161106231917-r21640-ge218f0f-efi.img.gz.sha256) [checksum](http://www.lakka.tv/doc/Verifying-MD5-Checksums) is correct ?

Thank you.

**Comment by maxengel** [comment]
I have confirmed the image is correct. The specific computer is running an Intel Atom processor, if it helps.

**Comment by gouchi** [comment]
On which device you try to run Lakka ?

**Comment by maxengel** [comment]
It is a [Kangaroo Plus](http://www.kangaroo.cc/kangarooplus/)

**Comment by gouchi** [comment]
Can you try to make a test with latest release [64](http://sources.lakka.tv/nightly/Generic.x86_64/Lakka-Generic.x86_64-devel-20161221183631-r21729-g274b40f-efi.img.gz) and [32](http://sources.lakka.tv/nightly/Generic.i386/Lakka-Generic.i386-devel-20161221183902-r21729-g274b40f-efi.img.gz) bit version? Also, you can try to use [Etcher](https://etcher.io) to create the usb stick.

Thank you.

**Comment by maxengel** [comment]
Yes, I'll try it with Etcher. Is there a reason the 64 bit release wouldn't work?

**Comment by gouchi** [comment]
No it should work. Did you [disable secure boot](http://forum.kodi.tv/showthread.php?tid=245335&pid=2185490#pid2185490) ?

**Comment by maxengel** [comment]
I did disable both secure boot and TPM.
